tests/test_dsp_pad_zeros.py

```diff
@@ -56,3 +56,66 @@ def test_pad_zeros():
         axis=-1)
 
     np.testing.assert_allclose(padded.time, desired)
+
+
+def test_pad_zeros_complex():
+    num_zeros = 2
+    n_samples = 8
+
+    test_signal = pyfar.signals.impulse(
+        n_samples, delay=0, amplitude=np.ones((2, 3)), sampling_rate=44100)
+    test_signal.complex = True
+
+    # test error raising for invalid modes
+    with pytest.raises(ValueError, match="Unknown padding mode"):
+        pyfar.dsp.pad_zeros(test_signal, 1, mode='invalid')
+
+    # test padding at the beginning of the signal
+    padded = pyfar.dsp.pad_zeros(test_signal, num_zeros, mode='beginning')
+    # check of dimensions are maintained
+    assert test_signal.cshape == padded.cshape
+    # check if final number of samples after padding is correct
+    assert test_signal.n_samples + num_zeros == padded.n_samples
+    # check if results if still complex
+    assert padded.complex is True
+
+    desired = pyfar.signals.impulse(
```

Comment:
Might be possible to remove all tests below since they are already tested above.